src/content/developers/tutorials/how-to-write-and-deploy-an-nft/index.md

```diff
@@ -149,10 +149,11 @@ Now time that our environment is set up, onto more exciting stuff: *writing our
 
 Open up the my-nft project in your favorite editor (we like [VSCode](https://code.visualstudio.com/)). Smart contracts are written in a language called Solidity which is what we will use to write our MyNFT.sol smart contract.‌
 
-1. Navigate to the “contracts” folder and create a new file called MyNFT.sol
+1. Navigate to the `“contracts”` folder and create a new file called MyNFT.sol
```

Comment:
Just the back-ticks will do, no need for " "... but while on this line, could extend that to 
```
1. Navigate to the `contracts` folder and create a new file called `MyNFT.sol`
```
1. Navigate to the `contracts` folder and create a new file called `MyNFT.sol`